src/gotcha.c

```diff
@@ -89,7 +99,8 @@ int prepare_symbol(struct internal_binding_t *binding)
       debug_printf(2, "Symbol %s found in %s at 0x%lx\n", 
                    user_binding->name, LIB_NAME(lib),
                    symtab[result].st_value + lib->l_addr);
-      setBindingAddressPointer(user_binding,(void *)(symtab[result].st_value + lib->l_addr));
+      *(struct internal_binding_t**)user_binding->function_handle = user_binding->opaque_handle; // TODO: does this mean we don't need opaque?
```

Comment:
@mplegendre, right now both user_binding->function_handle and user_binding->opaque_handle point the same place. This means that (so long as that remains true from now til the end of time) we could get rid of opaque_handle. That parenthetical is a dangerous one. Thoughts?